docs/authoring_studies.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Authoring studies in Cumulus
+_parent: Overview
+nav_order: 1
+# audience: clinical researcher, unfamiliar with the project
+# type: Explanation
+---
+# Introduction to studies in Cumulus
+
+Cumulus studies aim to help enable address the following kinds of research
+questions:
+
+- How can I compute a phenotype from the data in my EHR? And how would I apply
+that phenotype so I could conduct a clinical study?
+- How do I sample a cohort from my institution's patient population?
+- How do I make my work repeatable at another institution?
+
+By providing a set of tools to help answer these questions, we aim to
+accelerate the time from clinical question to actionable answer based on data
+you already have. After the initial configuration to get the Cumulus framework
+up and running, you can reuse much of that work for future research questions.
+
+We've published several papers using Cumulus, to give you the idea of the work
+the project enables. You can see the full list on the 
+[study List](https://docs.smarthealthit.org/cumulus/library/study-list.html)
+- each study contains a link to the associated paper.
```

Comment:
Why is this a list item? That draws visual attention to it that I'm not sure it deserves. :smile:

Maybe it could just be a sentence in the paragraph above, after the link to the study list?

Reply:
oh that was supposed to be a generic hyphen as a separator - i'll escape that.